### PR TITLE
update  sriov related failed cases

### PR DESCRIPTION
**Pull request by zhaozhanqi**
There is project nodeport-upgrade already exist in `features/upgrade/sdn/service-upgrade.feature`.  so change it 
@anuragthehatter ^^

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
and also update some sriov related failed.

**Comment by zhaozhanqi** [comment]
@anuragthehatter since your already updated the project name issue in another PR,  Here I removed the update,  Now only sriov related features are updated.

**Comment by rbbratta** [comment]
line 230 seems to have an undefined step?

```
    And admin checks that the "intel-netdevice-rhcos" sriov_network is deleted from the "openshift-sriov-network-operator" project 
```

File: features/networking/sriov.feature [hunk 1]
```diff
@@ -403,14 +403,13 @@ Feature: Sriov related scenarios
     When I execute on the pod:
       | bash | -c | cat /sys/class/net/net1/address |
     Then the step should succeed
-    And evaluation of `@result[:response].strip` is stored in the :pod1_net1_mac clipboard
+    And evaluation of `@result[:response].match(/\h+:\h+:\h+:\h+:\h+:\h+/)[0].strip` is stored in the :pod1_net1_mac clipboard
```
Comment (liangxia) [inline features/networking/sriov.feature:406]: `:response` contains both `:stdout` and `:stderr`.
You can use `:stdout` to get rid of `:stderr` msg.

Comment (zhaozhanqi) [reply features/networking/sriov.feature]: Thanks, updated.

**Pull request merged**